lib/combine_pdf/page_methods.rb

```diff
@@ -63,15 +63,16 @@ def inject_page(obj, top = true)
       # injecting each of the values in the injected Page
       res = resources
       obj.resources.each do |key, new_val|
```

Comment:
That may be. I wanted to mess with it as little as possible given my limited understanding and my lack of a functioning test suite.